jinahub/indexers/SimpleIndexer/__init__.py

```diff
@@ -57,8 +58,8 @@ def index(self, docs: 'DocumentArray', parameters: Dict, **kwargs):
         :param docs: the docs to add
         :param parameters: the parameters dictionary
         """
-        traversal_path = parameters.get('traversal_paths', self.default_traversal_paths)
-        flat_docs = docs.traverse_flat(traversal_path)
+        traversal_paths = parameters.get('traversal_paths', self.default_traversal_paths)
```

Comment:
Let's make this code robust to having `None` as `docs` inputs since `docs` is an Optional argument

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/47/commits/cfcfe9cfdea788b19822ba1ab53f480f7667c143) for the change.